srcalc/src/main/java/gov/va/med/srcalc/db/RiskModelDao.java

```diff
@@ -60,6 +67,11 @@ public RiskModel getRiskModelForId( final int mid )
         return (RiskModel)getCurrentSession().get(RiskModel.class, mid);
     }
     
+    /**
+     * Persist the given risk model to the database.
```

Comment:
This documentation is lacking, particularly in the difference between the passed object and the returned object. I would model this comment on `VariableDao.mergeVariable()`.